verifier/src/evidence.rs

```diff
@@ -125,7 +138,10 @@ mod test {
     use p256::ecdsa::VerifyingKey;
     use x509_cert::der::DecodePem;
 
-    fn collateral(tcb_info: &str) -> Collateral {
+    const TCB_INFO_JSON: &str = include_str!("../data/tests/fmspc_00906ED50000_2023_05_10.json");
+    const QE_IDENTITY_JSON: &str = include_str!("../data/tests/qe_identity.json");
+
+    fn collateral(tcb_info: &str, qe_identity: &str) -> Collateral {
```

Comment:
I'm hoping this will be limited to 2 inputs and that I can provide a test Certificate verifier which implements a certificate chain verifier Trait to force failures on certificate verification.